src/app/app.component.spec.ts

```diff
@@ -16,16 +16,10 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'angular-project'`, () => {
-    const fixture = TestBed.createComponent(AppComponent);
-    const app = fixture.componentInstance;
-    expect(app.title).toEqual('angular-project');
-  });
-
-  it('should render title', () => {
+  it('should render header', () => {
     const fixture = TestBed.createComponent(AppComponent);
```

Comment:
const fixture = TestBed.createComponent(AppComponent); better to move to beforeall/before each as you use it in each spec case.
tests should include just some custom logic and expectation, the enviroment creation and common things for all test cases should be moved in before each, before all blocks